### PR TITLE
Reduce screenshot area in footer snapshots

### LINKED ISSUE #328: Reduce screenshot area in snapshot tests
Since the viewport for snapshot tests is set to 1920px &times; 1080px &times; 2dpi each snapshot comes out to be a bitmap image of 3840px &times; 2160px. These images are too huge
- to store in the repository and track changes over time
- to compare in every run of the test

With most of the space being blank white anyways, it seems like a huge waste of storage and computing power to run these tests. A simpler way would be to just screenshot the node instead of the whole viewport.

Better approach: [Element screenshot](https://pptr.dev/#?product=Puppeteer&version=v3.0.0&show=api-elementhandlescreenshotoptions)

Current approach: [Page screenshot](https://pptr.dev/#?product=Puppeteer&version=v3.0.0&show=api-pagescreenshotoptions)

### PR DESCRIPTION
**Pull request by megha070**
## Fixes

Fixes #328 by @dhruvkb 

## Description
Reduce screenshot area in footer snapshots by taking screenshot of the node instead of the whole viewport.
## Tests
npm run test:snapshot
If changes are made, so as to update the screenshots use below command
npm run test:snapshot -- -u

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: tests/snapshot/footer.test.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ describe('Footer', () => {
   it('Default', async () => {
     await goTo(page, 'patterns-footer--default-story')
     await pause(1000)
-    const image = await page.screenshot()
-
-    expect(image).toMatchImageSnapshot()
+    const image = await page.$('footer.main-footer')
```
Comment (dhruvkb) [inline tests/snapshot/footer.test.js]: Just `footer` as the selector would also be okay.

**Review by dhruvkb — COMMENTED** [review]
The code is correct, please make the changes across all tests in the same PR. Also update the snapshots so that the GitHub Action can automatically verify if your code works.

**Comment by megha070** [comment]
Yes, Sure

**Comment by dhruvkb** [comment]
This is already incorporated in the code.

**Pull request closed without merging**